src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -503,4 +514,59 @@ public void testCaseCreationErrorWithBadExtension() throws Exception {
                     .andExpect(status().isUnprocessableEntity());
         }
     }
+
+    @Test
+    public void testChangeFilter() throws Exception {
+        final String filter = "{\"type\":\"CRITERIA\",\"equipmentFilterForm\":{\"equipmentType\":\"BATTERY\",\"name\":\"test bbs\",\"countries\":[\"BS\"],\"nominalVoltage\":{\"type\":\"LESS_THAN\",\"value1\":545430,\"value2\":null},\"freeProperties\":{\"region\":[\"north\"],\"totallyFree\":[\"6555\"],\"tso\":[\"ceps\"]}}}";
+        final String name = "filter name";
+        mockMvc.perform(put("/v1/explore/filters/{id}",
+                FILTER_UUID)
+                .contentType(APPLICATION_JSON)
+                .content(filter)
+                .param("name", name)
+                .header("userId", USER1)
+        ).andExpect(status().isOk());
+    }
```

Comment:
it could be nice for all of those tests to check we are sending a "name updating" request AND an "element updating" request : 
var request = server.takeRequest(1000, TimeUnit.MILLISECONDS);
assertTrue(requests.stream().anyMatch(r -> r.getPath().matches("(...)")));

Reply:
Done.